core/server/api_container/server/startosis_engine/plan_yaml/plan_yaml_generator.go

```diff
@@ -22,38 +22,56 @@ const (
 	outputFutureRefType    = "output"
 )
 
-// PlanYaml is a yaml representation of the effect of an Instructions Plan or sequence of instructions on the state of the Enclave.
-type PlanYaml struct {
-	privatePlanYaml *privatePlanYaml
+// PlanYamlGenerator is a yaml representation of the effect of an Instructions Plan or sequence of instructions on the state of the Enclave.
```

Comment:
this might need a slight update given it's now (more accurately, as I understand) named as a generator